build.gradle

```diff
@@ -68,6 +68,7 @@ allprojects {
         runtimeClasspath.exclude group: "org.slf4j"
         runtimeClasspath.exclude group: "org.jetbrains.kotlin"
         runtimeClasspath.exclude group: "org.jetbrains.kotlinx"
+        runtimeClasspath.exclude group: "software.amazon.awssdk", module: "netty-nio-client"
```

Comment:
Did you accidentally remove the change?

Reply:
No - there's no need to explicitly exclude the netty dependency - by removing the `nett-nio-client` - its dependencies don't get resolved.